Stream_support/include/CGAL/IO/File_scanner_OFF.h

```diff
@@ -35,6 +35,165 @@
 #include <CGAL/Point_3.h>
 #include <CGAL/Vector_3.h>
 
+
+inline CGAL::Color getIndexedColor(int id)
```

Comment:
This function name does not follow the naming rules of CGAL. As you are here modifying CGAL headers, and not Qt demos, there is no ambiguity: the naming convention is the convention of CGAL, and not the one of Qt.

Have a look at the developers manual: http://doc.cgal.org/latest/Manual/devman_code_format.html#secnaming_scheme

What is more, it is completely forbidden, to add a global function in the global namespace, in a CGAL header. That function would better be a private static function of the `File_scanner_OFF` class.